packages/build/tests/time/tests.js

```diff
@@ -2,10 +2,14 @@
 
 const { version } = require('process')
 
+const { intercept, cleanAll } = require('@netlify/nock-udp')
 const test = require('ava')
 
 const { runFixture } = require('../helpers/main')
-const { startUdpServer } = require('../helpers/udp_server')
+
+test.after(() => {
+  cleanAll()
+})
```

Comment:
One thing to note: this only works inside a single process. Some of `@netlify/build` call it through the CLI entry point (as opposed to programmatically). In that case, a new process would be used which would not have this global variable (`Socket.prototype`) monkey-patched.
However, in this specific file, this is not the case, so this is ok :+1:

Reply:
Yeah. We could add a comment within the file to safeguard anyone from trying to use the `runFixture` via CLI within this file?

Reply:
Yes, this might be good enough.
Using the CLI is much slower, so most tests are run programmatically. The only reason the CLI needs to be used is when testing features that are only available in the CLI. However, we've been trying to ensure that most features available in the CLI are also available programmatically (and vice-versa), so those tests are only for things like "Test --help flag" or "Test exit code". So, overall, it probably is not a big issue.